### PR TITLE
Updated the organisation section on cloud with some more information

### PR DESCRIPTION
**Pull request by jonat123**
- Added Payment history
- Added Subscriptions

### CODE REVIEW

File: Umbraco-Cloud/Getting-Started/The-Umbraco-Cloud-Portal/Organizations/index.md [hunk 1]
```diff
@@ -46,3 +46,11 @@ In the **Access Rights** section, you can get a list of all the Access Rights yo
...
 
 ![Payment methods](images/payment-methods-v10.png)
+
+## Payment History
+
+In the **Payment History** section, you can see the payment history for your organisation.
+
+## Subscriptions
+
+In the **Subscriptions** section you can see the current active subscribtions that is running under the Umbraco Cloud organisation.
```
Comment (eshanrnh) [inline Umbraco-Cloud/Getting-Started/The-Umbraco-Cloud-Portal/Organizations/index.md:56]:
```suggestion
In the **Subscriptions** section, you can see the current active subscriptions that are running under the Umbraco Cloud organization.
```

File: Umbraco-Cloud/Getting-Started/The-Umbraco-Cloud-Portal/Organizations/index.md [hunk 2]
```diff
@@ -46,3 +46,11 @@ In the **Access Rights** section, you can get a list of all the Access Rights yo
 In the **Payment methods** section, you can view the payment methods for your organization. From here, you can add or delete credit card details for your Organization.
 
 ![Payment methods](images/payment-methods-v10.png)
+
+## Payment History
+
+In the **Payment History** section, you can see the payment history for your organisation.
```
Comment (eshanrnh) [inline Umbraco-Cloud/Getting-Started/The-Umbraco-Cloud-Portal/Organizations/index.md:52]:
```suggestion
In the **Payment History** section, you can see the payment history for your organization.
```

**Review by eshanrnh — CHANGES_REQUESTED** [review]
Looks good, @jonat123 Just some minor changes needed 😀

**Comment by jonat123** [comment]
@eshanrnh perfect :D 

Suggestions applied!

Merging it in :D

**Pull request merged**